dashboards/nginx-ingress/nginx-ingress-prometheus.json

```
@@ -0,0 +1,198 @@
+{
+  "category": "CUSTOM",
+  "displayName": "NGINX Ingress Controller",
+  "mosaicLayout": {
+    "columns": 12,
+    "tiles": [
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "blankView": {},
+            "timeSeriesQuery": {
+              "prometheusQuery": "round(sum(irate(nginx_ingress_controller_requests[2m])), 0.001)"
+            }
+          },
+          "title": "Controller Request Volume"
+        },
+        "width": 4,
+        "xPos": 0,
+        "yPos": 0
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "sparkChartView": {
+              "sparkChartType": "SPARK_LINE"
+            },
+            "timeSeriesQuery": {
+              "prometheusQuery": "sum(avg_over_time(nginx_ingress_controller_nginx_process_connections[2m]))\n"
+            }
+          },
+          "title": "Controller Connections"
+        },
+        "width": 4,
+        "xPos": 4,
+        "yPos": 0
+      },
+      {
+        "height": 2,
+        "widget": {
+          "scorecard": {
+            "sparkChartView": {
+              "sparkChartType": "SPARK_LINE"
+            },
+            "timeSeriesQuery": {
+              "prometheusQuery": "sum(nginx_ingress_controller_success{controller_pod=~\".*\",controller_class=~\".*\",controller_namespace=~\".*\"})"
+            }
+          },
+          "title": "Config Reloads"
+        },
+        "width": 4,
+        "xPos": 8,
+        "yPos": 0
+      },
+      {
+        "height": 3,
+        "widget": {
+          "title": "Ingress Request Volume",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "round(sum(irate(nginx_ingress_controller_requests{controller_pod=~\".*\",controller_class=~\".*\",controller_namespace=~\".*\",ingress=~\".*\"}[2m])) by (ingress), 0.001)"
+                }
+              }
+            ],
+            "timeshiftDuration": "0s",
+            "yAxis": {
+              "label": "y1Axis",
+              "scale": "LINEAR"
+            }
+          }
+        },
+        "width": 6,
+        "xPos": 0,
+        "yPos": 5
+      },
+      {
+        "height": 3,
+        "widget": {
+          "title": "Ingress Status Codes",
```

In the screenshot, this is displaying as "Ingress Request Rate" which looks correct based on the query.

```suggestion
          "title": "Ingress Request Rate",
```